src/app/common/DataOverview.vue

```diff
@@ -445,25 +423,6 @@ function getRowAttributes({ name }: any): Record<string, string> {
 .with-warnings {
   color: var(--yellow-500);
 }
-
-.data-table-action-link {
-  padding: 0;
-}
-
-.action-link__active-state {
-  --size: 18px;
-
-  display: block;
-  width: var(--size);
-  height: var(--size);
-  line-height: var(--size);
-  border-radius: 50%;
-  background-color: var(--logo-green);
-  margin: 0 0 0 var(--spacing-xxs);
-  color: var(--white);
-  font-size: 13px;
-  text-align: center;
-}
```

Comment:
What's all this change/removal?

Reply:
I couldn’t find any use of this column. I think we used to highlight the selected row with some sort of green check mark icon, but that’s not present anymore in any view.

Reply:
Gotcha ty!